### PR TITLE
bugfix: dashboard status label right align

### PR DESCRIPTION
**Pull request by tmikeschu**
image

Diff coming from prettier

### CODE REVIEW

File: src/app/dashboard/components/dashboard.component.html [hunk 1]
```diff
@@ -2,42 +2,57 @@
...
+                    <mat-label class="contentlabel"
+                      >{{ request.name }}<br />
+                      <span class="content-multiplier">x</span
+                      ><span class="contentlabel-detail">{{ formatItemDetails(request) }}</span></mat-label
                     >
-                    <mat-label class="contentlabel">{{ request.name }}<br>
-                      <span class="content-multiplier">x</span><span
-                        class="contentlabel-detail">{{ formatItemDetails(request) }}</span></mat-label>
-                    <mat-label [className]="getStyle(request.status)" style="margin-left: auto;">
+                    <mat-label [className]="getStyle(request.status)" style="margin-left: auto; text-align: right;">
```
Comment (tmikeschu) [inline src/app/dashboard/components/dashboard.component.html:40]: this is the meaningful change

File: src/app/dashboard/components/dashboard.component.html [hunk 2]
```diff
@@ -2,42 +2,57 @@
...
+                [routerLink]="['/dashboard-item', 'request', request.itemId]"
+                class="contentborder dashboard-list-item"
+              >
+                <div
+                  class="swipe-container"
+                  [@.disabled]="request.status !== status.OrderCancelled && request.status !== status.OrderFulfilled"
+                >
+                  <div class="dashboardlistitem" [@rowAnimate]="request.rowState">
+                    <mat-label class="contentlabel"
+                      >{{ request.name }}<br />
```
Comment (jasonbrandt42) [inline src/app/dashboard/components/dashboard.component.html]: Can we keep tag elements to a same line as much as possible, breaking every line makes it hard to read where the tags end and start...

**Review by jasonbrandt42 — APPROVED** [review]

File: src/app/dashboard/components/dashboard.component.html (hunk 2 above)
Comment (tmikeschu) [reply src/app/dashboard/components/dashboard.component.html]: Hm it must be a prettier thing. I'll see if there is a fix  for that.

**Comment by tmikeschu** [comment]
@jasonbrandt42 I reverted the commit with prettier changes. Now it's just the style change and some whitespace cleanup.

The bracket thing seems like a [known issue](https://github.com/prettier/prettier/issues/6460). My opinion would be to stomach it for now and wait for the fix, but that's a separate convo for sure.

**Pull request merged**